app/misc.py

```diff
@@ -913,11 +913,11 @@ def getPostList(baseQuery, sort, page):
             posts = baseQuery.order_by(hot.desc()).limit(100).paginate(page, 25)
         elif config.database.engine == 'SqliteDatabase':
             posts = baseQuery.order_by(
-                (SubPost.score * 20 + (fn.datetime(SubPost.posted, 'unixepoch') - 1134028003) / 1500).desc()).limit(
+                (SubPost.score * 20 + (fn.Unix_Timestamp(SubPost.posted) - 1134028003) / 1500).desc()).limit(
                 100).paginate(page, 25)
         else:
             posts = baseQuery.order_by(
-                (SubPost.score * 20 + (fn.Unix_Timestamp(SubPost.posted) - 1134028003) / 1500).desc()).limit(
+                (SubPost.score * 20 + (fn.datetime(SubPost.posted, 'unixepoch') - 1134028003) / 1500).desc()).limit(
```

Comment:
This is breaking mysql support

Reply:
oops didn't mean to commit that one - I have to do this every time to make it work locally for me lol :sweat_smile: